### DISCUSSION TITLE
MIMIC-III Demo linking to waveform data

### DISCUSSION THREAD

**Discussion by Khaled-Mohammed-Abdelgaber** [MIT-LCP/mimic-code#1498] [MIMIC-III]
Hello team,
I have downloaded MIMIC-III demo files and I can not figure out how to match between subject id and its corresponding waveforms on physioBank. in physioBank divided into folders (from part 1 to part 9), each folder is divided into sub-folders and each subfolder is divided into multiples of records
I hope I will find the answer here.
Thanks in advance.

**Comment by alistairewj** [comment]
You can link the `subject_id` to the patient identifier in the waveform database. You must used the matched waveform database subset to make this link, and not every patient will have a waveform.
